src/nile/core/init.py

```diff
@@ -1,30 +1,11 @@
 """Command to kickstart a Nile project."""
 import logging
-import subprocess
-import sys
 from distutils.dir_util import copy_tree
 from pathlib import Path
 
-from nile.core.install import install
```

Comment:
shouldn't we also remove this module?

Reply:
Yes! I wanted to make sure the changes were accepted before deleting (and potentially adding it back). Will delete